src/main/java/tutoraid/logic/commands/DeleteStudentFromLessonCommand.java

```diff
@@ -16,69 +18,93 @@
 
 
 /**
- * Deletes a student from a lesson in TutorAid.
+ * Deletes students from lessons in TutorAid.
  */
 public class DeleteStudentFromLessonCommand extends DeleteCommand {
 
     public static final String COMMAND_FLAG = "-sl";
 
-    public static final String MESSAGE_USAGE = COMMAND_FLAG + ": Delete a student from a lesson "
-            + "by the student index number used in the last student list "
-            + "and the lesson index number used in the last lesson listing.\n"
-            + "Parameters: "
-            + PREFIX_STUDENT + "STUDENT_INDEX (must be a positive integer) "
-            + PREFIX_LESSON + "LESSON_INDEX (must be a positive integer) "
-            + "Example: " + COMMAND_FLAG
-            + " s/1"
-            + " l/2";
+    public static final String MESSAGE_USAGE = String.format("%1$s %2$s: Removes a student from a lesson."
```

Comment:
Should change `Removes a student from a lesson.` to `Removes student(s) from lesson(s).` to be more accurate